python/python/glide/async_commands/server_modules/json.py

```diff
@@ -139,6 +139,57 @@ async def get(
     return cast(bytes, await client.custom_command(args))
 
 
+async def arrlen(
+    client: TGlideClient,
+    key: TEncodable,
+    path: Optional[TEncodable] = None,
+) -> Optional[TJsonResponse[int]]:
+    """
+    Retrieves the length of the array at the specified `path` within the JSON document stored at `key`.
+
+    See https://valkey.io/commands/json.arrlen/ for more details.
```

Comment:
Remove the link. We can add valid links later once docs published

Reply:
Yeah, I will remove them all